### PR TITLE
Amplicon workflow v2

### PR DESCRIPTION
**Pull request by jkokosar**

### CODE REVIEW

File: resolwe_bio/processes/amplicon/bwa_trim.yml [hunk 1]
```diff
@@ -0,0 +1,180 @@
+# ==========================================
+# Align reads and trimm resulting SAM output
+# ==========================================
+---
+
+- slug: align-bwa-trim
```
Comment (JenkoB) [inline resolwe_bio/processes/amplicon/bwa_trim.yml]: Why this can't be added to current bwa aligner? Is result SAM or BAM file?

Comment (jkokosar) [inline resolwe_bio/processes/amplicon/bwa_trim.yml]: It trims primers after aligment directly from SAM file, only than the file is converted to BAM.

Comment (mstajdohar) [inline resolwe_bio/processes/amplicon/bwa_trim.yml]: Could this be an option in the current bwa aligner? For instance, trim primers if master file given.

File: resolwe_bio/processes/amplicon/bwa_trim.yml [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+  version: 0.0.1
+  type: data:alignment:bam:bwatrim
+  category: analyses
+  flow_collection: sample
+  persistence: CACHED
+  description: >
+    Align with BWA mem and trim the sam output.
+  input:
+    - name: master_file
+      label: Master file
```
Comment (mstajdohar) [inline resolwe_bio/processes/amplicon/bwa_trim.yml]: Provide a better description. Not obvious what this is for.

File: resolwe_bio/tools/plotcoverage.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python2
```
Comment (mstajdohar) [inline resolwe_bio/tools/plotcoverage.py]: We should start using Python 3.

**Comment by mstajdohar** [comment]
Fix tests

**Pull request merged**